tests/functional/configured_endpoint_urls/test_configured_endpoint_url.py

```diff
@@ -63,6 +66,22 @@ def create_cases():
         )
 
 
+def get_create_client_args(client_config):
+    create_client_args = {}
+
+    if client_config.get('endpoint_url', None):
```

Comment:
For these checks, let's just do an `in` comparator (e.g. `if 'endpoint_url' in client_config:`). Mainly if the value in the test case is falsey then it would skip the check which could be possible with the `ignore_configured_endpoint_urls` configuration value.